docs/user-guide/prereqs.md

```diff
@@ -13,6 +13,15 @@ If you downloaded a pre-compiled [release](https://github.com/kubernetes/kuberne
 
 If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
 
+If you don't have the source code checked out you can download the compiled kubectl binary for each release with curl
+
+```shell
+# Linux
+curl -LO https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl
+# OS X
+curl -LO https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/darwin/amd64/kubectl
+```
+
 The kubectl binary doesn't have to be installed to be executable, but the rest of the walkthrough will assume that it's in your PATH.
 
 The simplest way to install is to copy or move kubectl into a dir already in PATH (e.g. `/usr/local/bin`). For example:
```

Comment:
That's true. I thought about making the whole command 1 line like the [minikube release docs show](https://github.com/kubernetes/minikube/releases) but wanted to reuse as much existing docs as possible.
Would it be better just to have 3 distinct ways in the docs to install kubectl?

1. From release
2. From git clone
3. From compiling binary